server/models/BookingRequest.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require("mongoose");
+
+const bookingRequestSchema = new mongoose.Schema(
+  {
+    userId: {
+      type: mongoose.Schema.Types.ObjectId,
+      required: true,
+      ref: "User",
+    },
+    sitterId: {
+      type: mongoose.Schema.Types.ObjectId,
+      required: true,
+      ref: "User",
+    },
+    start: {
+      type: Date,
+      default: Date.now,
+      required: true,
+    },
+    end: {
+      type: Date,
+      default: Date.now,
+      required: true,
+    },
+    description: {
+      type: String,
+      default: "",
+    },
+    accepted: {
```

Comment:
```suggestion
    payment: {
```
Please change this field to 'payment' to keep track of payment status